client/src/import_utils.ts

```diff
@@ -0,0 +1,119 @@
+/* eslint @typescript-eslint/triple-slash-reference: "off" */
+// CGQAQ: Without next line the test will fail, using import won't work
+/// <reference path="./types/vscode-cache.d.ts" />
+
+import got from "got";
+import VC from "vscode-cache";
+
+interface ModuleInfo {
+  name: string;
+  description: string;
+  star_count: string;
+  search_score: number;
+}
+
+export async function searchX(keyword: string): Promise<ModuleInfo[]> {
+  // https://api.deno.land/modules?limit=10&query=$QUERY
+  const response: {
+    success: boolean;
+    data: {
+      total_count: number;
+      results: ModuleInfo[];
+    };
+  } = await got(
+    `https://api.deno.land/modules?limit=20&query=${keyword}`
```

Comment:
You can actually increase this limit to 100 if it helps. In a later pass we could also use the `page` parameter to get a list of all modules.

Also can you url encode the keyword?